clients/ui/bff/cmd/main.go

```diff
@@ -53,8 +58,19 @@ func main() {
 
 	// Start the server in a goroutine
 	go func() {
-		logger.Info("starting server", "addr", srv.Addr)
-		if err := srv.ListenAndServe(); err != nil && err != http.ErrServerClosed {
+		logger.Info("starting server", "addr", srv.Addr, "TLS enabled", (certFile != "" && keyFile != ""))
+		var err error
+		if certFile != "" && keyFile != "" {
+			// Configure TLS if both cert and key files are provided
```

Comment:
- Shall we do something like:
```golang
      if _, err := os.Stat(certFile); err != nil {
            logger.Error("TLS certificate file error", "path", certFile, "error", err)
            os.Exit(1)
        }
        if _, err := os.Stat(keyFile); err != nil {
            logger.Error("TLS key file error", "path", keyFile, "error", err)
            os.Exit(1)
        }
````
To make it clear if the files are provided via flag but don't exist? Or the errors throw by srv.ListenAndServe() are enough?